modules/sdk-coin-zketh/src/zketh.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @prettier
+ */
+import { BaseCoin, BitGoBase, MPCAlgorithm } from '@bitgo/sdk-core';
+import { BaseCoin as StaticsBaseCoin, coins } from '@bitgo/statics';
+import { AbstractEthLikeNewCoins, TransactionBuilder as EthLikeTransactionBuilder } from '@bitgo/abstract-eth';
+import { TransactionBuilder } from './lib';
+
+export class Zketh extends AbstractEthLikeNewCoins {
+  protected constructor(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>) {
+    super(bitgo, staticsCoin);
+  }
+
+  static createInstance(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>): BaseCoin {
+    return new Zketh(bitgo, staticsCoin);
+  }
+
+  protected getTransactionBuilder(): EthLikeTransactionBuilder {
+    return new TransactionBuilder(coins.get(this.getBaseChain()));
+  }
+
+  /** @inheritDoc */
+  supportsTss(): boolean {
+    return false;
+  }
```

Comment:
just double checking, why doesn't this support tss? I thought all new eth likes were tss enabled.

Reply:
As per the current scope of the project, we do not support TSS wallets for zkSync. Also, there is nothing planned for the near future.